src/moscot/plotting/_utils.py

```diff
@@ -401,3 +425,36 @@ def _plot_temporal(
     if save:
         fig.figure.savefig(save, bbox_inches="tight")
     return fig
+
+
+def _color_transition(c1: str, c2: str, num: int, alpha: float) -> List[str]:
+    if not mpl.colors.is_color_like(c1):
+        raise ValueError(f"{c1} cannot be interpreted as an RGB color.")
+    if not mpl.colors.is_color_like(c2):
+        raise ValueError(f"{c2} cannot be interpreted as an RGB color.")
+    c1_rgb = np.array(mpl.colors.to_rgb(c1))
+    c2_rgb = np.array(mpl.colors.to_rgb(c2))
+    return [mpl.colors.to_rgb((1 - n / num) * c1_rgb + n / num * c2_rgb) + (alpha,) for n in range(num)]
+
+
+def _create_col_colors(adata: AnnData, obs_col: Optional[str], subset: Optional[str]) -> Optional[mcolors.Colormap]:
+    if obs_col is not None and subset is not None:
+        if isinstance(subset, list) and isinstance(subset[0], str):
```

Comment:
Base on the typehint, this can never happen.

Reply:
Would either pass the responsibility to the caller, or modify the signature + change the check `if isinstance(subset, Sequence) and not isinstance(subset, str): subset = subset[0]`